Library/Homebrew/software_spec.rb

```diff
@@ -107,6 +108,10 @@ def resource(name, klass = Resource, &block)
     end
   end
 
+  def get_resource
+    @resource
+  end
```

Comment:
Attribute readers in Ruby don't normally have a `get_` prefix. Is there any reason we cannot name this `resource` and replace the entire method with a simple `attr_reader :resource`.